learn/wavemaker-release-notes/v10-7-0.md

```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
 - `getLib()`: Returns the library used for the widget.
 - `overrideDefaults(options)`: Customize the options which the library supports.
 
+### Backend Enhancements
+
+- For Rest/Swagger services, the rest call goes from the backend when you enable proxy, making the direct path empty in service definitions.
+- Introduced the Allowed host property in the security tab of the profile configuration. It prevents host header injections and lets application developers control the allowed hosts of the application.
```

Comment:
Prevents HTTP Host header injection attacks. Developers can specify the host names where the WaveMaker application is deployed and platform will rejects HTTP requests which have different values in Host header than this list.